stronk/schemas/workout/query.py

```diff
@@ -0,0 +1,34 @@
+
+import graphene
+
+from stronk.models.program_workouts import ProgramWorkouts as ProgramWorkoutsModel
+from stronk.models.workout import Workout as WorkoutModel
+
+from stronk.schemas.workout.type import Workout as WorkoutType
```

Comment:
Would you be ok with sticking to `Workout` to make it consistent with the user and exercise graphql queries/mutations?

Reply:
yup that works!